internal/cloud/cloud.go

```diff
@@ -526,7 +527,7 @@ func gitIgnorePaths(projectDir string) ([]string, error) {
 
 	fpath := filepath.Join(projectDir, ".gitignore")
 	if _, err := os.Stat(fpath); err != nil {
-		if os.IsNotExist(err) {
+		if errors.Is(err, os.ErrNotExist) {
```

Comment:
Yeah, but in package `os` there's `ErrNotExist   = fs.ErrNotExist   // "file does not exist"`. I think there's no need to import `fs` when `os` is imported. And as you can see in the code, it uses `ErrNotExist` instead of `fs.ErrNotExist`. WDYT?

Reply:
My understanding is that this `os` code is kinda legacy, and new code should we using the `io/fs` APIs and constants. The golang team explain their motivations in this design document: https://go.googlesource.com/proposal/+/master/design/draft-iofs.md

They've left the code in `os` package as a temporary measure to not break existing code of golang users. However, eventually, they'll more formally deprecate these `os` APIs for filesystem usage, and after a period remove them. 

This is why I'd prefer the code we write to be using the newer APIs....it feels a step backwards to use the older APIs/constants.